docs/knowledge_base/tips/long_callbacks/index.md

```diff
@@ -0,0 +1,73 @@
+# Long Running Callbacks
+
+Tasks in web applications can take a lot of time, which can lead to problems with communication between 
+the server (Python application) and the client (web browser).
+
+![Long Running Callbacks](long_running_callbacks.png){width=100%}
+
+To deal with this issue, Taipy has a helpful feature called "long-running callbacks" to address this problem. 
+These callbacks allow the server to do demanding tasks in the background while still making sure 
+the user interface stays quick to respond. 
+This article talks about the idea of long-running callbacks in Taipy, gives examples of how to use them, 
+and shows how they make the user experience better.
+
+## Example 1: Executing a Heavy Task in the Background
+
+Imagine a situation where a callback starts a task that requires a lot of resources and time to finish. 
+To make this work, we can use a straightforward approach:
+
+![Executing a Heavy Task in the Background](long_running_callbacks_2.png){width=100%}
+
+In the previous example, the Taipy function called `invoke_long_callback()` manages the resource-intensive task. 
+It sets up a separate background thread to run the `heavy_function()`, allowing the rest of the application to continue running. 
+The `on_action` function gets activated by a user action, like clicking a button, for instance.
+
+## Example 2: Monitoring Task Status
```

Comment:
```suggestion
# Example 2: Monitoring Task Status
```

Reply:
"\`heavy_function()\`" => "\*heavy_function()\*"
"The \`on_action\` function" -> "The `on_action\` callback"

Reply:
Wrong line comment, no?